app/navigation/Navigator.scala

```diff
@@ -469,6 +468,12 @@ class Navigator @Inject() () {
       case None    => WhatIsYourRoleAsImporterController.onPageLoad(NormalMode, userAnswers.draftId)
       case Some(_) => RequiredInformationController.onPageLoad(userAnswers.draftId)
     }
+  private def contactsNextPage(userAnswers: UserAnswers): Call             = {
+    val isAgent =
+      false // fixme fetch from userAnswers once Martin G has added the field (ARSSTB-160)
+    if (isAgent) ProvideTraderEoriController.onPageLoad(userAnswers.draftId)
+    else CheckRegisteredDetailsController.onPageLoad(NormalMode, userAnswers.draftId)
+  }
```

Comment:
This could be handled by the various UserRoles now

Reply:
I've had to revert an attempt to integrate with UserRoles - it can't support the individual case - the UserRoleProvider returns an exception if it isn't an agent or employee of org.